### ISSUE TITLE
Missing requirement for pydantic 2 in setup.py

### ISSUE THREAD

**Issue by bcxpro** [imryche/blockkit#80]
Hey there,

I wanted to give you a heads up about an issue I've come across with the recent release of Blockkit 1.6.0.

It seems like Blockkit 1.6.0 doesn't have a version constraint for its dependency on pydantic. Even though it seems to need pydantic 2, the version on PyPI doesn't specify this requirement. So, when I tried to use Blockkit 1.6.0 along with pydantic 1 in my project, pip upgraded to 1.6.0 but kept pydantic 1. Unfortunately, Blockkit 1.6.0 doesn't work properly with pydantic 1.

I noticed that the setup.py file doesn't mention any requirement for pydantic 2. As a result, package managers think it's okay to install Blockkit 1.6.0 alongside pydantic 1, which leads to the compatibility problem I ran into.

Could you consider adding a constraint to the package that requires pydantic 2? 

Thanks so much for looking into this.

Cheers,

**Comment by imryche** [comment]
Hey! Thanks for bringing this to my attention.

I've added a version constraint for pydantic, requiring version 2 or higher. Feel free to upgrade to 1.6.1, and you should no longer encounter this compatibility issue.

**Comment by bcxpro** [comment]
Thanks,

   My workaround for the issue was this (in poetry):

``
blockkit = "^1.5.1,!=1.6.0"
pydantic = "^1.9.0"
```

That is, banning 1.6.0.  Now that you published 1.6.1 with the right constraints the dependency resolver installs 1.5.2 instead of 1.6.1 and everything works.

However, if I unban 1.6.0

``
blockkit = "^1.5.1"
pydantic = "^1.9.0"
```

Then the dependency resolver installs 1.6.0 because it is the latest version "compatible" with pydantic 1.

So my guess is that you have to "[yank](https://blog.piwheels.org/new-features-deletion-yanking-and-more/)" version 1.6.0 in pypi.

Cheers,

**Comment by imryche** [comment]
Hi, I've decided to simply remove version 1.6.0 from PyPI. This should resolve the issue.
